mdflynn/rancid-tomatillos
# Feature/final styling

## What is the change?
- Add PropTypes to each file
- Adds styling to the main page
- Finishes overall styling for the App
- Adds an additional unit test

## What does it fix?
- No fix the Pull Request

## Is this a fix or a feature?
This is a feature

## Where should the reviewer start?
- The bottom of each component file should be reviewed to ensure proper PropTypes were added
- Each file has had formatting updates, check each file for formatting consistency
- Lines 16-36 in Movie.js

## How should this be tested?
Two ways: 
- Run `npm test` in your terminal to see a passing test suite
- Clone down this repo
  - Cd into directory
  - Run `npm start` then navigate to localhost:3000 in your browser to see current site
